TombEngine/Specific/savegame/schema/ten_savegame.fbs

```diff
@@ -368,6 +368,9 @@ table ParticleInfo {
 	fx_obj: int32;
 	room_number: int32;
 	node_number: int32;
+	targetPos: Vector4;
+	SpriteSeqID: GAME_OBJECT_ID;
```

Comment:
Flatbuffers specification requires all fields to be in a lowercase, and do not use CamelCase. So instead of `SpriteSeqID`, it should be `sprite_seq_id`, and instead of `targetPos`, it should be `target_pos`.